lib/composed.ts

```diff
@@ -1,10 +1,12 @@
 'use strict'
 
-const ObservableStore = require('.')
+import ObservableStore = require('.')
```

Comment:
What's this syntax? 👀

Reply:
[This is the import for the corresponding `export =` syntax.](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)

Reply:
💡